### ISSUE TITLE
Servers without domains should appear in listings

### ISSUE THREAD

**Issue by Zehra** [bzlist/bzlist.net#8]
**Describe the bug**
Servers which don't have/use domains aren't listed.

**To Reproduce**
1. View "Public Servers Online"
2. Search for a public server which does not use a domain.
3. Find no servers which fit the category.

**Expected behavior**
Finding servers which do not use a domain if there is any.

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
Basically in short, the list server provides the following info:
`domain:port (stuff) ipAddress`
`domain` can also be an IP Address, but in the case that it is, we get matching data, both the "`domain`" and "`ipAddress`" are the same. (I may later check to see how it's handled.)

**Comment by The-Noah** [comment]
This has been fixed
